en/identity-server/next/docs/guides/your-is/recover-username.md

```diff
@@ -0,0 +1,23 @@
+# Recover your username
+
+If you are the owner or an administrator of an WSO2 Identity Server organization, the following guide explains how you can recover a forgotten username.
+
+To recover your username:
+
+1. Go to the sign-in page of the [My Account Portal]({{base_path}}/guides/your-is/is-self-service/#access-my-account-portal).
+
+    ![Recover your username]({{base_path}}/assets/img/guides/organization/self-service/customer/recover-your-username.png){: width="300" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+
+2. Click **Forgot username?**.
+
+3. Enter the information for the account you’re trying to recover, then click **Next**.
+
+    ![Username recovery claiim]({{base_path}}/assets/img/guides/organization/self-service/customer/username-recovery-claim.png){: width="300" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
```

Comment:
```suggestion
    ![Username recovery claim]({{base_path}}/assets/img/guides/organization/self-service/customer/username-recovery-claim.png){: width="300" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
```

Reply:
Fixed by https://github.com/wso2/docs-is/pull/4858/commits/7c797952be0d1b954991ed2889ed15992b0564b4